TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/views.py

```diff
@@ -26,27 +33,68 @@ def pretty_request(request, data):
     )
 
 
-# TODO implement stub with backend
 @api_view(['POST'])
 @parser_classes([JSONParser])
-def start_historical_migration(request):
+def osi_create_migration(request):
     request_data = request.data
-    status_code = 200
     logger.info(pretty_request(request, request_data))
+
+    osi_serializer = OpenSearchIngestionCreateRequestSerializer(data=request_data)
+    osi_serializer.is_valid(raise_exception=True)
+    migration = OSIMigrationLogic()
+    try:
+        migration.create_pipeline_from_json(input_json=request_data, pipeline_template_path=PIPELINE_TEMPLATE_PATH)
+    except InvalidAuthParameters as i:
+        logger.error(f"Error performing osi_create_migration API: {i}")
+        return Response(str(i), status=status.HTTP_400_BAD_REQUEST)
+    except Exception as e:
+        logger.error(f"Error performing osi_create_migration API: {e}")
+        return Response(str(e), status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+
     response_data = {
-        'Timestamp': datetime.datetime.now(datetime.timezone.utc)
+        'Timestamp': datetime.datetime.now(datetime.timezone.utc),
+        'MigrationType': MigrationType.OSI_HISTORICAL_MIGRATION
```

Comment:
They haven't and in its current context isn't super useful. I originally was going to have this function not be specific to a certain type of migration and this would have been more useful.